main.py

```diff
@@ -1,10 +1,53 @@
-import init_django_orm  # noqa: F401
+import json
+
+from django.db.utils import IntegrityError
 
+import init_django_orm  # noqa: F401
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player in data:
```

Comment:
Iterate through items

Reply:
this way?
`for player, details in data.items():`